docs/guide/markdown.md

```diff
@@ -192,98 +192,6 @@ export default defineUserConfig({
 })
 ```
 
-It can be used in combination with the other marks below. Please leave a space between them.
-
-#### Line Highlighting
```

Comment:
I think it still worth introducing the features here, cuz it's quite useful features, and most themes should have implemented it.

We may only need to update the config reference link and add notes for theme authors